datasets/huishoudelijkafval/tabellen/bagobjectloopafstand.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
Shouldn't this be:
`datasets/huishoudelijkafval/tabellen/2.0.0/bagobjectloopafstand.json`?

Reply:
Not sure it matters, there is only one version of that table right now, right?

Reply:
That's for now, but what about future?

Reply:
If a new version of the table is added, the files can be moved anywhere, as long as the $ref in dataset.json is updated everything will work. Dataset.json contains info on the versions. In the future if we want to publish nondefault versions of schemas we will need to put more thought into how this is going to work exactly at that time. I didnt want to get ahead on that more than neccessary now. Creepy scopes and all.

Reply:
Fair, though current location can be confusing (given we're still in discussion about what and how).